config/application.rb

```diff
@@ -144,7 +144,11 @@ class Application < Rails::Application
     )
 
     # Check that the loggers setting exist to configure the app and sql loggers
-    Foreman::Logging.add_loggers(SETTINGS[:loggers] || {:app => {:enabled => true}, :sql => {:enabled => true}})
+    Foreman::Logging.add_loggers(SETTINGS[:loggers] || {
+      :app => {:enabled => true},
+      :sql => {:enabled => true},
+      :crs => {:enabled => true}
```

Comment:
Should we default this to off, since it may log a lot?  It's easy to enable now from settings.yaml if needed.